webapp/src/websocket.tsx

```diff
@@ -3,10 +3,19 @@ import {Dispatch} from 'redux';
 import {GetStateFunc} from 'mattermost-redux/types/actions';
 import {WebSocketMessage} from 'mattermost-redux/types/websocket';
 
+import {fetchNote} from 'client';
 import {setNoteValue} from 'redux_connectors';
 
 export function handleWebsocketNoteUpdate(_: GetStateFunc, dispatch: Dispatch) {
     return async (msg: WebSocketMessage<{note: string}>) => {
         dispatch(setNoteValue(msg.broadcast.channel_id, msg.data.note));
     };
 }
+
+export function handleWebsocketChannelViewed(_: GetStateFunc, dispatch: Dispatch) {
```

Comment:
Not sure the websocket is the right move here. Seems prone to failure and delay? Could we just mount a root component that could handle the logic for this? Or have a store subscription like the RHS logic for playbooks does?

Reply:
Yes, agree. I'll go the root component way.